jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -24,13 +26,15 @@ class SubsetPlugin(TemplateMixin):
     template_file = __file__, "subset_plugin.vue"
     select = List([]).tag(sync=True)
     subset_items = List([]).tag(sync=True)
-    subset_selected = Unicode("Create new").tag(sync=True)
+    subset_selected = Unicode("Create New").tag(sync=True)
     mode_selected = Unicode('add').tag(sync=True)
     show_region_info = Bool(True).tag(sync=True)
     subset_types = List([]).tag(sync=True)
     subset_definitions = List([]).tag(sync=True)
     has_subset_details = Bool(False).tag(sync=True)
 
+    is_editable = Bool(False).tag(sync=True)
+
```

Comment:
is it possible to replace `is_editable` with checking the length of `subset_defintions` or are there cases where the logic needs to be different (single regions that will show in the plugin but are not considered editable)?

(I first checked to see if `has_subset_details` does the same thing, but that seems to never actually be used, so maybe that can be removed here as well).

Reply:
> single regions that will show in the plugin but are not considered editable

Yes, masked subset state is not editable.

Reply:
Or if in the future, we add support for drawing weird shapes but has no translator for it, that is not editable too with the current logic.